js/features/supportFileUploads.js

```diff
@@ -45,6 +45,13 @@ export function handleFileUpload(el, property, component, cleanup) {
 
     el.addEventListener('change', eventHandler)
 
+    // If the Livewire property has changed to null or an empty string, then reset the input
+    component.$wire.$watch(property, (value) => {
+        if (value === null || value === '') {
+            el.value = ''
```

Comment:
We can use "clearFileInputValue()" here right? (if we extract it to a util fuction?)